earth_enterprise/src/fusion/rasterfuse/TmeshGenerator.h

```diff
@@ -37,18 +36,19 @@ class TmeshGenerator
   std::vector<etVec3f>      elev;
 
  public:
-  TmeshGenerator(const khTilespaceFlat &sampleTilespace_,
-                 bool decimate_) :
-      sampleTilespace(sampleTilespace_),
-      decimate(decimate_)
```

Comment:
Why take out from constructor ?

Reply:
It's now being passed in to the Generate() function directly along with decimation_threshold

Reply:
As far as keeping the decimate value in the constructor goes, I genuinely don't see the point. Can you explain please? It is not being used from there. As 0.009 is now no longer hardcoded into the Generate() function, we are now calling Generate() with decimation_threshold value regardless.